modules/comments/base.php

```diff
@@ -277,18 +277,4 @@ public function set_comment_cookies( $comment_id ) {
 			setcookie( 'comment_author_url_'   . COOKIEHASH, esc_url($comment->comment_author_url), time() + $comment_cookie_lifetime, COOKIEPATH, COOKIE_DOMAIN );
 		}
 	}
-
-	/**
```

Comment:
Instead of creating a new function, I wonder if we could add the `graph.facebook.com` domain to the list of banned domains in Photon, here:
https://github.com/Automattic/jetpack/blob/master/functions.photon.php#L303

That would solve the issue everywhere where such URLs are being used, not just in comments.

That said, it would be interesting to find out why the domain was whitelisted in the first place:
https://github.com/Automattic/jetpack/blame/master/functions.photon.php#L275

It seems it was done here:
https://code.trac.wordpress.org/ticket/17

cc @mdawaffe